services/appmaker/controller.go

```diff
@@ -32,12 +32,26 @@ func (s *server) Create(ctx context.Context, body *pb.RequestBody) (*pb.Response
 	if err != nil {
 		return nil, err
 	}
+	
+	filter := types.M{
+		"owner": body.GetOwner(),
+		"instance_type": "application",
+		"date": time.Now().Format("2006-02-01"),
```

Comment:
Dont hardcode key values. define them as constants. If we ever require to change them then we dont need to change these keys at places in the code.

Reply:
done